lib/shims/redis-shim.ts

```diff
@@ -54,8 +54,8 @@ export default class RedisShim {
         await isolatedClient.watch(key);
         await isolatedClient
           .multi()
-            .unlink(key)
-            .hSet(key, data)
+          .unlink(key)
```

Comment:
I'd like to keep this indented as it make sense in the context the sense of how MULTI and EXEC is used in Redis.

Reply:
That was my linter's fault, sry about it. I have vscode to lint on save